gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-core/src/main/java/io/gravitee/am/gateway/handler/root/resources/endpoint/mfa/MFAEnrollEndpoint.java

```diff
@@ -277,8 +278,7 @@ private void saveEnrollment(RoutingContext routingContext) {
             // redirect to the original request
             redirectToAuthorize(routingContext);
         } else {
-            // parameters are invalid
-            routingContext.fail(400);
+            routingContext.fail(new EnrollmentChannelValidationException("Validation failed"));
```

Comment:
**nickpick:** not sure about the message here :thinking: but I do not have any better idea than "Invalid Parameters" :man_shrugging: